erisdb/params.go

```diff
@@ -90,13 +90,6 @@ type (
 		GasLimit int64  `json:"gas_limit"`
 	}
 
-	// Used when sending a 'Send' transaction.
-	SendParam struct {
```

Comment:
because in TMSP refactor `Send` and `SendAndHold` had already been removed, and in the merge this struct redundantly lingered (noticed when I manually ran over the diff).  Arguably this will be reverted if `Send` and `SendAndHold` are not to be deprecated, but to not add to the amount of loose ends, I choose to for now also delete it.

In https://github.com/eris-ltd/eris-db/issues/115 we can discuss what the fate of `Send` and `SendAndHold` is.